sdl2/chase_in_space/chase_in_space.odin

```diff
@@ -0,0 +1,229 @@
+/*
+	Simple game about enemies chasing and shooting you in 2D space.
+
+	Original repo: https://github.com/wbogocki/Odin-Play
+
+	To play:
+		- Move with WSAD or arrow keys
+		- Jump with Shift + direction
+*/
+
+package chase_in_space
+
+import "core:fmt"
+import "core:math"
+import "vendor:sdl2"
+
+Game :: struct {
+	renderer: ^sdl2.Renderer,
+	keyboard: []b8,
```

Comment:
Why use `b8` when `GetKeyboardStateAsSlice` returns `u8` on purpose?

Reply:
It's because if they were `u8` they have to be cast to bool or explicitly compared to 0 in lines like this:
```odin
if game.keyboard[sdl2.SCANCODE_UP] | game.keyboard[sdl2.SCANCODE_W] do dir.y -= 1
```

So a b8 makes these lines a bit shorter and just overall matches better with how these are/would be used. Should I still change them?